mods/d2k/rules/aircraft.yaml

```diff
@@ -1,46 +1,98 @@
-^CARRYALL:
+carryall.scripted:
 	Inherits: ^Helicopter
 	Valued:
 		Cost: 1200
 	Tooltip:
 		Name: Carryall
 		Description: Large winged, planet-bound ship\n  Automatically lifts harvesters.
-	Buildable:
-		Queue: Aircraft
-		BuildPaletteOrder: 120
 	Health:
 		HP: 250
 	Armor:
 		Type: Light
 	Helicopter:
-		CruiseAltitude: 2100
+		CruiseAltitude: 2048
 		InitialFacing: 0
 		ROT: 4
 		Speed: 160
 		LandableTerrainTypes: Sand, Rock, Transition, Spice, Dune
-		RepairBuildings: repaira,repairo,repairh
-		RearmBuildings: starporta,starporto,starporth
+		RepairBuildings: repair
+		RearmBuildings:
 		Repulsable: False
-		LandAltitude: 100
+		LandAltitude: 512
 		LandWhenIdle: False
+	RenderUnit:
+		Image: carryall
 	WithShadow:
 	LeavesHusk:
-		HuskActor: CARRYALL.Husk
+		HuskActor: carryall.husk
 	-Selectable:
+
+carryall:
+	Inherits: carryall.scripted
 	AutoCarryall:
+	Buildable:
+		Queue: Aircraft
+		BuildPaletteOrder: 120
+	Helicopter:
+		CruiseAltitude: 2100
+		LandAltitude: 100
 
-FRIGATE:
+carryall.infantry:
+	Inherits: ^Plane
 	ParaDrop:
-		DropRange: 1c0
+		DropRange: 5c0
+		ChuteSound:
+	Health:
+		HP: 200
+	Armor:
+		Type: Light
+	RevealsShroud:
+		Range: 12c0
+	Plane:
+		ROT: 4
+		Speed: 280
+		RepairBuildings: repair
+		RearmBuildings:
+		Repulsable: False
+	RenderUnit:
+		Image: carryall
+	WithShadow:
+	Cargo:
+		MaxWeight: 5
+		Types: Infantry
+	-Selectable:
+	-GainsExperience:
+	Tooltip:
+		Name: Carryall
+	LeavesHusk:
+		HuskActor: carryall.infantry.husk
+	RejectsOrders:
+
+carryall.husk:
+	Inherits: ^AircraftHusk
+	Tooltip:
+		Name: Carryall
+	Helicopter:
+		ROT: 4
+		Speed: 210
+		RepairBuildings: repair
+		RearmBuildings:
```

Comment:
The default values hardcode RA-specific buildings.  These overrides are needed to clear those.